components/script/dom/bluetoothremotegattcharacteristic.rs

```diff
@@ -108,7 +105,7 @@ impl BluetoothRemoteGATTCharacteristicMethods for BluetoothRemoteGATTCharacteris
                 instance_id
             } => {
                 Ok(BluetoothRemoteGATTDescriptor::new(self.global().r(),
-                                                      &self,
+                                                      self,
```

Comment:
In my opinion, it seems that the & isn't necessary. Maybe because the function gets &self already. Of course there's a high possibility that I'm wrong.